extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -4,14 +4,14 @@
 
 
 .i-amphtml-story-system-layer {
-  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.32), transparent) !important;
+  background: linear-gradient(180deg, #0004, #0002 33%, #0001 67%, #0000) !important;
   position: absolute !important;
   display: flex !important;
   justify-content: space-between !important;
   top: 0 !important;
   left: 0 !important;
   right: 0 !important;
-  height: 54px !important;
+  height: calc(var(--story-page-vh) * 15) !important;
```

Comment:
If the icons aren't responsive, why should the height here be?

Reply:
The plan is to make them responsive but that requires extra testing so didn't want to get all that in the same PR (and I already had this numbers matching the specs). I can leave this line out for when they become responsive, but I'd still need to update the height to be taller than 54px (as per the mocks).

Reply:
Updating the height but keeping it fixed sounds good, let's keep the size scaling for when we prioritize it